js/close-big-photo.js

```diff
@@ -0,0 +1,20 @@
+import { bigPictureContainer } from './create-big-photo.js';
+const bigPictureCloseButton = document.querySelector('.big-picture__cancel');
+
+const closePhoto = () => {
+  bigPictureContainer.classList.add('hidden');
+  document.body.classList.remove('modal-open');
+};
+
+const keyDownEsc = (evt) => {
+  if (evt.code === 'Escape') {
+    closePhoto();
+  }
+};
+
+function closeBigPhotoPopup() {
+  document.addEventListener('keydown', keyDownEsc);
```

Comment:
```suggestion
  document.addEventListener('keydown', documentKeydownHadler);
  или
  document.addEventListener('keydown', onDocumentKeydown);
```

https://up.htmlacademy.ru/javascript/23/criteries#d4

> Из названия обработчика события и функции-колбэка следует, что это обработчик.

выбери как тебе нравится называть, через хендлер или через он и так всё и называй

Reply:
все остальные поправь